public/js/src/client.js

```diff
@@ -366,7 +383,7 @@ $(document).ready(function(){
 			}
 		}
 	}
-
+// This is a factory. Generally in JS, they're capitalized, and I end my var names with the word 'Factory,' because I'm anal and wordy.
```

Comment:
A factory (in the Java context in which I am familiar) just means a (usually static) peice of code which returns an object of a specific type in a specific way.  Is that the same context here?